doc/en/user/source/services/features/service.rst

```diff
@@ -0,0 +1,89 @@
+Using OGC API - Features service
+--------------------------------
+
+The OGC API Features Service is accessed via the :guilabel:`Features` version :guilabel:`1.0.1` link on the home page.
+
+.. figure:: img/welcome-ogc-api-features.png
+   
+   GeoServer Welcome Page OGC API - Features
+
+Capabilities
+''''''''''''
+
+The service is self described using:
+
+* ``html``: A collection of web pages, with links for navigation between content (and that can be indexed by search engines for discoverability).
+
+  .. figure:: img/features.png
+ 
+     OGC API Features service
+
+* `application/json`: A collection of :file:`json` documents, with reference between each document for programmatic access by web developers.
+
+  .. code-block:: json
+  
+     {
+       "title": "GeoServer Features services",
+       "description": "This services delivers vector data in raw form, including both geometries and attributes.",
```

Comment:
This services -> This service? (singular)

Reply:
Oh interesting point, this is the title and description proposed for ``wfs.xml`` as part of GSIP-202. The heading + description makes more sense when describing several versions of WFS, but not so much for the single version of OGCAPI - Features.

Should be either:

* ``These services deliver vector data in raw form, including both geometries and attributes``
* ``This service delivers vector data in raw form, including both geometries and attributes``

I will correct for now but we can revist data directory default another time.